asapdiscovery-alchemy/asapdiscovery/alchemy/cli.py

```diff
@@ -250,6 +249,14 @@ def gather(network: str, allow_missing: bool):
     network_with_results.to_file("result_network.json")
 
 
+def validate_traceback_flag(ctx, param, value):
+    """Validate traceback flag --with-traceback is only used in conjunction with --errors flag."""
+    if value:
+        if not ctx.params.get("errors"):
+            raise click.UsageError("--with-traceback requires --errors flag to be set.")
+    return value
```

Comment:
Perhaps instead of forcing users to use both `-et`, `-t` could imply `-e`? In that sense, `-t` is a stronger form of `-e`?

Reply:
Yeah, I think that makes sense, I'll make this change.